### PR TITLE
Adds option to set the className on the input field

### PR DESCRIPTION
**Pull request by ianwitherow**
Allows you to specify the className for the input field. I needed this since I'm using Bootstrap, so I can set the input class to "form-control", etc. Defaults to the original class name of "input-calendar-value".

### CODE REVIEW

File: src/Calendar.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ module.exports = React.createClass({
         maxDate: React.PropTypes.any,
         format: React.PropTypes.string,
         inputFieldId: React.PropTypes.string,
+		  inputFieldClass: React.PropTypes.string,
```
Comment (Rudeg) [inline src/Calendar.js]: please fix indent

File: dist/Calendar.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ module.exports = React.createClass({displayName: "exports",
         maxDate: React.PropTypes.any,
         format: React.PropTypes.string,
         inputFieldId: React.PropTypes.string,
+		  inputFieldClass: React.PropTypes.string,
```
Comment (Rudeg) [inline dist/Calendar.js]: please fix indent

**Comment by Rudeg** [comment]
Hey, please fix intends and I'll merge it

**Comment by ianwitherow** [comment]
Will do, sorry, it looked normal in my editor. Which do you prefer? I see a mixture of 2 spaces and 4. Looks like 4 spaces is more common.

**Comment by ianwitherow** [comment]
Should be good. I think vim was having issues because of the two different styles so I had to go about it a little differently.

**Comment by Rudeg** [comment]
Cool, thanks!

**Pull request merged**